app/Fission/Internal/Orphanage/RIO.hs

```diff
@@ -71,3 +59,6 @@ instance
         |> mkClientEnv manager
         |> runClientM query
         |> liftIO
+
+instance MonadTime (RIO cfg) where
+  currentTime = liftIO getCurrentTime
```

Comment:
This entire module is going to get burned down in a future PR where we'll introduce the long awaited `newtype Fission a = Fission { unFission :: RIO Config a }`